P2/task_2.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Created by yahor.turets on 17.07.2017.
+ */
+var DateDisplayFormatter;
```

Comment:
According to task object must contain methods to parse ticks, ms;
Also none of utility methods implemented: like point 5 from task 2